tests/resources/resolveimports/forkedImport.cellml

```diff
@@ -0,0 +1,13 @@
+<?xml version="1.0" encoding="iso-8859-1"?>
+<model name="forked_import"
+	xmlns="http://www.cellml.org/cellml/2.0#"
+	xmlns:cellml="http://www.cellml.org/cellml/2.0#"
+	xmlns:xlink="http://www.w3.org/1999/xlink">
+	<import xlink:href="prong.cellml">
+		<component name="left" component_ref="component1" />
+	</import>
+	<import xlink:href="prong.cellml">
+		<component name="right" component_ref="component1" />
+	</import>
+	<component name = "centre"/>
+</model>
```

Comment:
Replace tabs with spaces (1 tab = 2 spaces).

Reply:
Same in some other test CellML files.

Reply:
cbb69365